### ISSUE TITLE
Team does not provide HTMLURL

### ISSUE THREAD

**Issue by kplimack** [google/go-github#2088]
The teams package provides `URL` which points at the api url, but not `HTMLURL` which is provided via api

when getting a team it will supply

```json
   "html_url": "https://github.com/orgs/github/teams/justice-league"
```

**Comment by kplimack** [comment]
IMO, better off providing it vs having people parse/mutate it on their own.

**Comment by gmlewis** [comment]
Fixed by #2244.
Closing.
